src/custom/components/swap/EthWethWrap/index.tsx

```diff
@@ -122,7 +123,9 @@ const ButtonWrapper = styled.div`
 
 const ErrorMessage = ({ error }: { error: Error }) => (
   <ErrorWrapper>
-    <strong>{error.message}</strong>
+    <strong>
+      <Trans>{error.message}</Trans>
```

Comment:
Also don't think it'll be translatable at build time

Reply:
also true